docker-compose.yml

```diff
@@ -159,12 +171,19 @@ services:
   postgres:
     image: "postgres:16.0-alpine3.18"
     environment:
-      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
```

Comment:
Why has this been removed?

Reply:
good question
1. because replaced by `POSTGRES_PASSWORD=${PG_DB_PASS:?err}` which has the same value
2. because i confess I don't get the difference between the default root password and the password on the database that is created for the user

probably if the value of PG_DB_PASS was different from the one from POSTGRES_PASSWORD, I could investigate and better understand myself